src/metabase/premium_features/token_check.clj

```diff
@@ -320,8 +320,10 @@
     "Get the features associated with the system's premium features token."
     []
     (try
-      (or (some-> (premium-embedding-token) valid-token->features)
-          #{})
+      (or
+       #{"table-data-editing"}
```

Comment:
Just noticed this while passing through, guess it was for debugging. I'm sure you would remove it when not WIP but leaving a comment just in case.